src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -410,7 +430,11 @@ public bool ImportLibrary(string library)
                 OnLibraryLoadFailed(new LibraryLoadFailedEventArgs(library, e.Message));
                 return false;
             }
+
             OnLibraryLoaded(new LibraryLoadedEventArgs(library));
+
+            // After a library is loaded, update the library core data with the liveRunner core data
+            UpdateLibraryCoreData();
```

Comment:
Since you have stuck this in here in `ImportLibrary` have you verified that all of Dynamo initialization, library loading, package installs, passes thro' here?